services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceHelixAdmin.java

```diff
@@ -5330,6 +5415,10 @@ private OfflinePushStatusInfo getOfflinePushStatusInfo(
       // Check whether cluster is in maintenance mode or not
       if (isClusterInMaintenanceMode(clusterName)) {
         moreDetailsBuilder.append("Cluster: ").append(clusterName).append(" is in maintenance mode");
+      } else if (StringUtils.isNotEmpty(targetedRegions)) {
+        moreDetailsBuilder.append("Version for topic ")
+            .append(kafkaTopic)
+            .append(" isn't created since this is a targeted region push");
```

Comment:
shouldnt we print this line only if target region != current region?

Reply:
If that's a targeted region, the `executionStatus` will not be `NOT_CREATED` so we don't need this check, though it's probably more accurate in case kafka has some issues